saga-core/src/main/java/io/servicecomb/saga/core/ForwardRecovery.java

```diff
@@ -25,9 +25,9 @@
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
   @Override
-  public void apply(SagaTask task, SagaRequest request) {
+  public int apply(SagaTask task, SagaRequest request) {
     boolean success = false;
-
+    int retryCount = 0;
```

Comment:
if retry count is only for testing, it's a bit overkill to change signature for this purpose.

you can verify how many times the task mock is invoked instead.

Reply:
Done